overlord/servicestate/quota_control.go

```diff
@@ -32,19 +37,82 @@ import (
 	"github.com/snapcore/snapd/systemd"
 )
 
+var (
```

Comment:
nitpick, this var declaration can be moved into the same block as the systemdVersionError declaration